server/cli/src/refresh_dates.rs

```diff
@@ -39,6 +39,20 @@ fn get_timestamp_fields() -> Vec<TableAndFieldName> {
     .collect()
 }
 
+#[cfg(feature = "postgres")]
+fn get_exclude_timestamp_fields() -> Vec<TableAndFieldName> {
```

Comment:
There is a test to compare pg schema vs fields specified for update, some fields like this one and probably document timestamps which are in hash, should not be refreshed (or influence refresh), thus added to exclusion (still need to have all datetime fields listed, but the are ignored in the refresh logic)